gnssanalysis/gn_io/sp3.py

```diff
@@ -35,10 +35,20 @@
 # File descriptor and clock
 _RE_SP3_HEAD_FDESCR = _re.compile(rb"\%c[ ]+(\w{1})[ ]+cc[ ](\w{3})")
 
+# Nodata ie NaN constants for SP3 format
+SP3_CLOCK_NODATA_STRING  = " 999999.999999" # Not used for reading, as fractional components are optional
```

Comment:
I was also motivated to do it because I think it makes it clearer what is being done when a value is e.g. used several layers into some calculation logic.